src/backend/wsl.ts

```diff
@@ -691,21 +691,44 @@ export default class WSLBackend extends events.EventEmitter implements VMBackend
     await this.wslInstall(trivyExecPath, '/usr/local/bin');
   }
 
-  protected async installGuestAgent(kubeVersion: semver.SemVer | undefined) {
+  protected async installGuestAgent(kubeVersion: semver.SemVer | undefined, cfg: RecursivePartial<BackendSettings>) {
+    let guestAgentConfig: Record<string, any>;
+    let privilegedServiceEnabled;
```

Comment:
`privilegedServiceEnabled` is never set to true. Is there supposed to be a config on this?